packages/pn-commons/src/types/AppResponse.ts

```diff
@@ -0,0 +1,113 @@
+export type HTTPStatusCode = number | string;
+
+export type AppResponseOutcome = 'success' | 'error';
+
+/**
+ * The following ServerResponse interfaces is used to model DTO
+ * received after any webapi call
+ */
+export interface ServerResponse {
```

Comment:
Verificare che le interfacce vengano effettivamente utilizzate all'esterno. In caso contrario rimuovere export

Reply:
ok